manifests/0000_31_cluster-baremetal-operator_06_clusteroperator.cr.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: config.openshift.io/v1
```

Comment:
This should probably come before the deployment, since the deployment uses it.

Reply:
I'm not sure whether it matters. Shouldn't CBO create the CO deployement if not found anyways> FYI, MAO uses the same runlevel.

Reply:
Based on the note in [this section of the docs](https://github.com/openshift/cluster-version-operator/blob/master/docs/dev/clusteroperator.md#how-clusterversionoperator-handles-clusteroperator-in-release-image) I think the ClusterOperator resource has to come after the Deployment. The CVO doesn't use the manifest to actually create the resource, it only uses it to know which CO resources it should wait for before continuing with the next step of the install/upgrade. So, I think this is correct.